### PR TITLE
Add Doxymentation to Off reading

### PR DESCRIPTION
**Pull request by jschlenger**
adding doxy comments to the classes that do the off file reading

### CODE REVIEW

File: Core/src/OffConverter/OffReader.h [hunk 1]
```diff
@@ -26,24 +26,83 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
 #include "OffModel.h"
 
 OIP_NAMESPACE_OPENINFRAPLATFORM_CORE_OFFCONVERTER_BEGIN
+
+/*! \brief The Off reader class
+	*
+	* This class is used to read the information from an .off file and pass it to the Off Model
+	*/
```
Comment (pjanck) [inline Core/src/OffConverter/OffReader.h]:
```suggestion
/*! \brief The Off reader class
 *
 * This class is used to read the information from an .off file and pass it to the Off Model
 */
```

We try to have the starts vertically aligned in Doxycomments.

File: Core/src/OffConverter/OffReader.h [hunk 2]
```diff
@@ -26,24 +26,83 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
...
+	//! destructor
 	virtual ~OffReader();
 
+	/**
+	* \brief Does all the reading and calls the other functions
+	*
+	* This is the function to call to read all the information from the .off file
+	*
+	* \param[in] filename The name of the .off file
+	*/
```
Comment (pjanck) [inline Core/src/OffConverter/OffReader.h]: `\return` is missing

File: Core/src/OffConverter/OffReader.h [hunk 3]
```diff
@@ -26,24 +26,83 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
...
+	* \param[in] vertices The list of vertices that was already red from the .off file.
+	* \param[in] offVertices The list of vertices read from the .off file meant for the vertex buffer.
+	*/
 	static void readQuadFace(std::stringstream& lineStream,
 		std::vector<uint32_t>& indices,
 		std::vector<buw::VertexPosition3Color3Normal3>& vertices,
 		std::vector<buw::Vector3f>& offVertices);
+
+	/**
+	* \brief Reads all the vertices from the .off file
```
Comment (pjanck) [inline Core/src/OffConverter/OffReader.h]:
```suggestion
	* \brief Reads and interprets all the vertices from the .off file
```

File: Core/src/OffConverter/OffReader.h [hunk 4]
```diff
@@ -26,24 +26,83 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
...
+
+	/**
+	* \brief Calculates the normal of a triangle
+	*
+	* This function calculates the normal based on a triangle given by three vertices.
+	*
+	* \param[in] vertex1 The first vertex of the triangle.
+	* \param[in] vertex2 The second vertex of the triangle.
+	* \param[in] vertex3 The third vertex of the triangle.
+	*/
```
Comment (pjanck) [inline Core/src/OffConverter/OffReader.h]: return is missing

**Review by pjanck — CHANGES_REQUESTED** [review]
Some missing content

File: Core/src/OffConverter/OffReader.h (hunk 4 above)
Comment (jschlenger) [reply Core/src/OffConverter/OffReader.h]: Resolved: 464a368

File: Core/src/OffConverter/OffReader.h (hunk 3 above)
Comment (jschlenger) [reply Core/src/OffConverter/OffReader.h]: Resolved: 464a368

File: Core/src/OffConverter/OffReader.h (hunk 2 above)
Comment (jschlenger) [reply Core/src/OffConverter/OffReader.h]: Resolved: 464a368

File: Core/src/OffConverter/OffReader.h (hunk 1 above)
Comment (jschlenger) [reply Core/src/OffConverter/OffReader.h]: Resolved: 464a368

**Pull request merged**